co-authors-plus.php

```diff
@@ -380,14 +383,15 @@ public function coauthors_meta_box( $post ) {
 				<?php
 				foreach ( $coauthors as $coauthor ) :
 					$count++;
+					$avatar_url = get_avatar_url( $coauthor->ID, array( 'default' => 'gravatar_default' ) );
 					?>
 					<li>
-						<?php echo get_avatar( $coauthor->user_email, $this->gravatar_size ); ?>
```

Comment:
We can still use `get_avatar()` here for display in the case that there is no JS.  I'd recommend passing in the coauthor `ID` instead of the `user_email` though since guest authors may not always have an email assigned.